docker-for-mac/release-notes.md

```diff
@@ -16,10 +16,40 @@ This page contains information about the new features, improvements, known issue
 > Starting with Docker Desktop 3.0.0, Stable and Edge releases are combined into a single release stream for all users. Updates to Docker Desktop will now be available automatically as delta updates from the previous version. This means, when there is a newer version of Docker Desktop, it will be automatically downloaded to your machine. All you need to do is to click **Update and restart** from the Docker menu to install the latest update.
```

Comment:
@mat007 This needs to be updated as well.

Reply:
Ah yes, good catch!
Should we just replace it with a 3.3.0 block? Or split it in two: one 3.0.0 and a new 3.3.0? The latter feels a bit too much…
@stephen-turner @usha-mandya what do you think?

Reply:
Yes, replacing with 3.3.0 behavior sounds good.

Reply:
I went with:
> Starting with Docker Desktop 3.0.0, Stable and Edge releases are combined into a single release stream for all users. Updates to Docker Desktop will be available automatically as delta updates from the previous version. When an update is available, Docker Desktop displays an icon to indicate the availability of a newer version. Whenever convenient, you can start the download of the update in the background. When the download finishes, all you need to do is to click **Update and restart** to install the latest update.

Reply:
Yes, I think it's fine to lose the banner now.